### PR TITLE
use testing app API to run occ commands in UI tests start script

### PR DESCRIPTION
**Pull request by individual-it**
## Description
As it's now possible to run `occ` through the testing app we want to use that possibility in the UI tests start script.
The script still needs the local `occ` command to enable the testing app. When testing a remote instance (e.g. inside docker) the testing app needs to be installed and enabled by other means and the ui test script has to be started with the  `--remote` switch

## Motivation and Context
The main goal is to make it possible to run UI tests against a remote instance, e.g. inside docker. See also owncloud/qa-enterprise#115
For that to happen we need to reduce the use of the local `occ` command.

## How Has This Been Tested?
running UI tests locally and in travis

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: tests/travis/start_ui_tests.sh [hunk 1]
```diff
@@ -18,21 +18,48 @@ verlt() {
...
 
+#@param $1 admin password
+#@param $2 occ url
+#@param $3 command
+#sets $REMOTE_OCC_STDOUT and $REMOTE_OCC_STDERR from returned xml date
+#@return occ return code given in the xml data
+remote_occ() {
+	RESULT=`curl -s -u admin:$1 $2 -d "command=$3"`
+	RETURN=`echo $RESULT | xmllint --xpath "string(ocs/data/code)" - | sed 's/ //g'`
+	#we could not find a propper return of the testing app, so something went wrong
```
Comment (phil-davis) [inline tests/travis/start_ui_tests.sh]: s/propper/proper

**Review by phil-davis — COMMENTED** [review]
text in a comment only

**Comment by individual-it** [comment]
changes made, please review again

**Comment by individual-it** [comment]
backport in #29765

**Review by phil-davis — APPROVED** [review]

**Pull request merged**